### ISSUE TITLE
The search drop down menu with Cards behind the embed app content in Teams app\web

### ISSUE THREAD

**Issue by Koniksan** [integrations/microsoft-teams#97]
**Describe the bug**
The search drop down menu with Cards behind the embed app content in Teams app\web

**Expected behavior**
Drop-down is visible with correct z-index property 
![image](https://user-images.githubusercontent.com/15952613/109493668-e11a8d80-7a9d-11eb-9981-d190a499f695.png)

**Screenshots**
Drop-down is not visible
![image](https://user-images.githubusercontent.com/15952613/109493473-9a2c9800-7a9d-11eb-90c5-2cb6b5ae1475.png)

**Comment by ashokirla** [comment]
@Koniksan Can you explain a bit more about this issue. Is this related to GitHub app notifications.

**Comment by Koniksan** [comment]
@ashokirla  No, this is not related with GitHub app notifications, it`s only about **css** UI styling, maybe I created this issue in a wrong place. If it is not the right place could you forward me to the appropriate repo

**Comment by gauravsaralMs** [comment]
@Koniksan : most probably this issue belongs to https://github.com/microsoft/AdaptiveCards

**Comment by Koniksan** [comment]
> @Koniksan : most probably this issue belongs to https://github.com/microsoft/AdaptiveCards

Thanks, I think this issue can be closed
